heltour/tournament/views.py

```diff
@@ -1439,6 +1439,10 @@ def view(self, post=False):
         leagues = [self.league] + list(
             League.objects.filter(is_active=True).order_by('display_order').exclude(
                 pk=self.league.pk))
+
+        player = Player.get_or_create(self.request.user.username)
+        slack_linked = bool(player.slack_user_id)
```

Comment:
You don't need to explicitly cast this, although it's not harmful. You could just pass this value into the context directly and the if statement would still work, and it would be considered slightly more pythonic, I think.